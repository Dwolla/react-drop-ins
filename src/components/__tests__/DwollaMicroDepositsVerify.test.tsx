import TestRenderer from 'react-test-renderer';
import { DwollaMicroDepositsVerify } from '../index';

describe('DwollaMicroDepositsVerify', () => {
  it('should render the component', () => {
    const tree = TestRenderer.create(
      <DwollaMicroDepositsVerify customerId="123" fundingSourceId="456" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
