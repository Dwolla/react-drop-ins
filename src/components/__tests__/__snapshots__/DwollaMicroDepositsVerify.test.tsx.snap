// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DwollaMicroDepositsVerify should render the component 1`] = `
<dwolla-micro-deposits-verify
  customerId="123"
  fundingSourceId="456"
/>
`;
